_includes/footer.html

```diff
@@ -1,58 +1,23 @@
-{% assign identifier = site.data.usa_identifier.identifier_data[0] %}
-<div class="usa-identifier">
-  <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier">
-    <div class="usa-identifier__container">
-      <div class="usa-identifier__logos">
-        <a href="https://www.gsa.gov/" class="usa-identifier__logo">  
-          {% asset {{identifier.agency_logo}} class="usa-identifier__logo" alt="{{ identifier.agency }} Logo" %}
-        </a>
+{% assign identifier = site.data.usa_identifier %}
 
+<footer class="usa-footer" role="contentinfo">
+<div class="usa-footer__secondary-section">
+  <div class="grid-container">
+    <div class="grid-row grid-gap">
+      <div class="usa-footer__contact-links mobile-lg:grid-col-12">
+        <div class="usa-footer grid-row">
+          <div class="grid-col-auto">
+            This project is maintained by <a class="usa-link" href="https://18f.gsa.gov/">18F</a>
```

Comment:
consider using `<a class="usa-link" href="{{ identifier. org_secondary_url }}">{{ identifier. org_secondary }}</a> `

Reply:
i like that we keep noticing more places we could/should be using these identifiers heh